scripts/run-cypress.sh

```diff
@@ -127,12 +127,15 @@ else
   export CYPRESS_MIGRATE=$CYPRESS_MIGRATE
 fi	
 
+echo "${RUN_SPECIFIC_TEST}"
```

Comment:
If you wanna keep this let's make it friendly

```suggestion
[ -n "${RUN_SPECIFIC_TEST}" ] && echo "Running ${RUN_SPECIFIC_TEST}" || echo "Running all tests"
```

Reply:
I see you resolved this since I removed it (I assume), but for documentation's sake I'll respond: this echo was really only a troubleshooting echo. I think the information cypress gives you when the tests execute (or fail to execute) is actually generally enough information, so I don't think it's worth keeping the echo around in a more friendly form.